tf/examples/Bonsai/bonsai_example.py

```diff
@@ -90,6 +89,7 @@ def main():
                                   learningRate, X, Y, useMCHLoss, outFile)
 
     sess = tf.InteractiveSession()
+    # sess.run(tf.global_variables_initializer())
```

Comment:
I've fixed this. It now reads as 
sess.run(tf.group(tf.global_variables_initializer(), tf.local_variables_initializer())).
I need to use tf.local_variables_initializer() because I use the function tf.metrics.mean_absolute_error() in BonsaiTrainer, and it internally uses two local variables, which have to be initialised. Without that line, one would be getting a 'Failed Pre-Condition Error'.